src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/CheckPileup.java

```diff
@@ -42,42 +43,60 @@
  *
  * <h3>Usage example</h3>
  * <pre>
- * ./gatk-launch CheckPileup \
- *   -R reference.fasta \
- *   -I input.bam \
- *   -O output.txt \
- *   --pileup samtools.pileup
+ *   gatk CheckPileup \
+ *     -R reference.fasta \
+ *     -I input.bam \
+ *     -O output.txt \
+ *     --pileup samtools.pileup
  * </pre>
  *
  */
+@DocumentedFeature
 @CommandLineProgramProperties(
     summary = "This tool compares the mpileup data (reference base, aligned base from each overlapping read, and quality score) " +
             "generated internally by GATK to a reference pileup data generated by Samtools, for each position in the requested " +
             "interval.",
     oneLineSummary = "Compare GATK's internal pileup to a reference Samtools mpileup",
     programGroup = QCProgramGroup.class)
-@DocumentedFeature
 public final class CheckPileup extends LocusWalker {
 
     /**
      * This is the existing mpileup against which we'll compare GATK's internal pileup at each genome position in the desired interval.
      */
-    @Argument(fullName = "pileup", shortName = "pileup", doc="Pileup generated by Samtools")
+    @Argument(
+            fullName = "pileup",
+            shortName = "pileup",
```

Comment:
if the short name and long name are the same you don't have to specify the short name

Reply:
done